safe-navigation-operator/test.ooc

```diff
@@ -20,5 +22,5 @@ raise(thirdDerivativeB power != 5, "thirdDerivativeB power is not 5")
 raise(thirdDerivativeC power != 5, "thirdDerivativeC power is not 5")
 
 // This should not crash
-thirdDerivativeD := Polynomial new(2) $ derivativeFunc() $ derivativeFunc() $ derivativeFunc() $ derivativeFunc() 
+thirdDerivativeD := Polynomial new(2) $ derivativeFunc() $ derivativeFunc() $ derivativeFunc() $ derivativeFunc()
 raise(thirdDerivativeD power != 0, "thirdDerivativeD power is not 0")
```

Comment:
This is still incorrect, thirdDerivativeD evaluates to null, not to a derivative with a power of 0.